catroid/src/main/res/values-ar/strings.xml

```diff
@@ -326,12 +326,12 @@
   <!-- Project activity view -->
   <string name="project_name">البرنامج:</string>
   <string name="my_projects_activity_title">البرامج</string>
-  <string name="number_of_objects">Objects:</string>
+  <string name="number_of_objects">Objects: %1$d</string>
   <string name="start_scene_name">%1$s (First Scene)</string>
-  <string name="number_of_scripts">النص:</string>
-  <string name="number_of_bricks">اللّبنات:</string>
-  <string name="number_of_looks">المشهد:</string>
-  <string name="number_of_sounds">الأصوات:</string>
+  <string name="number_of_scripts">النص: %1$d</string>
+  <string name="number_of_bricks">اللّبنات: %1$d</string>
+  <string name="number_of_looks">المشهد: %1$d</string>
+  <string name="number_of_sounds">الأصوات: %1$d</string>
```

Comment:
Maybe Chrome is just tricking me or that's how RTL languages work but this change looks odd.
To test the Pull Request, [CAT-2199](https://jira.catrob.at/browse/CAT-2199) needs to be fixed first.